tracing-appender/src/rolling.rs

```diff
@@ -107,6 +112,41 @@ impl io::Write for RollingFileAppender {
     }
 }
 
+/// Creates an minutely, rolling file appender.
+///
+/// The appender returned by `rolling::minutely` can be used with `non_blocking` to create
+/// a non-blocking, minutely file appender.
```

Comment:
I realize that this never actually states that this rotates the log file once per minute...can we add that, just in case folks are confused by "minutely"?

Reply:
Thanks for taking the time to review this! rolling::n_minutes() is a great idea, I'll save that for a future change.

I'll update the doc and re-run rustfmt, looks like I forgot to format the file correctly